File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/mariadb/IntegerTypesUnsigned.kt
package dev.typr.foundations.docs.mariadb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class IntegerTypesUnsigned {
    //start
    val unsignedTiny: MariaType<Uint1> = MariaTypes.tinyintUnsigned
    val unsignedInt: MariaType<Uint4> = MariaTypes.intUnsigned
    val unsignedBig: MariaType<Uint8> = MariaTypes.bigintUnsigned
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/mariadb/IntegerTypesUnsigned.scala
package dev.typr.foundationssc.docs.mariadb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object IntegerTypesUnsigned:
  // start
  val unsignedTiny: MariaType[Uint1] = MariaTypes.tinyintUnsigned
  val unsignedInt: MariaType[Uint4] = MariaTypes.intUnsigned
  val unsignedBig: MariaType[Uint8] = MariaTypes.bigintUnsigned
  // stop
